judge/ratings.py

```diff
@@ -125,13 +145,11 @@ def rate_contest(contest):
     rating_sorted = rating_subquery.order_by('-contest__end_time')
     users = contest.users.order_by('is_disqualified', '-score', 'cumtime', 'tiebreaker') \
         .annotate(submissions=Count('submission'),
-                  last_rating=Coalesce(Subquery(rating_sorted.values('rating')[:1]), 1200),
```

Comment:
I think we still need to keep this for rating floors and ceilings.

Reply:
(and if we do keep it, we might want to make `1200` a constant like `MEAN_INIT`).

Reply:
Now that I think about it, `last_rating__lt` might cause an error. I will fix this soon.

Reply:
Fixed, rating floors and ceilings no longer cause an error.